os/drivers/seclink/seclink_drv_utils.h

```diff
@@ -1,34 +1,81 @@
-#ifndef _SECLINK_DRIVER_UTILS_H__
-#define _SECLINK_DRIVER_UTILS_H__
+/****************************************************************************
+ *
+ * Copyright 2021 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+#pragma once
+
 #ifndef LINUX
 #include <debug.h>
 #endif
+#ifdef CONFIG_SECURITY_LINK_DRV_PROFILE
+#include <unistd.h>
+#include <tinyara/timer.h>
+#endif
 
 #ifndef LINUX
-#define SLDRV_LOG sevdbg
+#define SLDRV_LOG lldbg
 #else
 #define SLDRV_LOG printf
 #endif
 
 #define SLDRV_TAG "[SECLINK_DRV]"
 
-#define SLDRV_ERR(fd)										\
-	do {													\
-		SLDRV_LOG(SL_TAG"%s:%d ret(%d) code(%s))\n",		\
-				  __FILE__, __LINE__, fd, strerror(errno));	\
+#define SLDRV_ERR(fd)                                       \
+	do {                                                    \
+		SLDRV_LOG(SL_TAG "%s:%d ret(%d) code(%s))\n",       \
+				  __FILE__, __LINE__, fd, strerror(errno)); \
 	} while (0)
 
-#define SLDRV_ENTER											\
-	do {													\
-		SLDRV_LOG(SLDRV_TAG"%s:%d\n", __FILE__, __LINE__);	\
+#define SLDRV_ENTER
+
+#ifdef CONFIG_SECURITY_LINK_DRV_PROFILE
+
+#define SLDRV_CALL(ret, res, method, param)                                         \
+	do {                                                                            \
+		if (se->ops->method) {                                                      \
+			sldrv_timer_handle hnd;                                                 \
+			(void)sldrv_start_time(&hnd);                                           \
```

Comment:
If fail to open the timer, we don't need to call `sldrv_get_time` also.
How about checking the return of `sldrv_start_time` and doing next when success to open the timer?

Reply:
It's test code. So I think adding exception handling is not necessary.